src/components/data-table-v2/_data-table-v2-sort.scss

```diff
@@ -66,3 +66,118 @@
     min-width: 14px;
   }
 }
+
+@mixin data-table-v2-sort--x {
+  // -------------------------------------
+  // Sortable table
+  // -------------------------------------
+
+  .#{$prefix}--data-table-v2--sort th:hover {
+    background: $data-table-column-hover;
+    border-top: 1px solid $data-table-column-hover;
+    border-bottom: 1px solid $data-table-column-hover;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-icon:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-menu:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-expand-v2:hover {
+    background: $ui-03;
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2--sort th {
+    padding: 0;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-checkbox {
```

Comment:
Should this end up being a 48x48 square? Seems like it's rendering un-evenly:

![image](https://user-images.githubusercontent.com/3901764/54783113-89572480-4bee-11e9-8c01-c2f7a0b20d51.png)

Reply:
This is something I fix, it's coming from a style from the checkbox label. But I didn't include it because I thought that would go in the select PR.